hazelcast-client/src/main/java/com/hazelcast/client/proxy/ClientReplicatedMapProxy.java

```diff
@@ -93,7 +93,7 @@ public ClientReplicatedMapProxy(String serviceName, String objectName) {
     @Override
     protected void onInitialize() {
         int partitionCount = getContext().getPartitionService().getPartitionCount();
-        targetPartitionId = ThreadLocalRandom.current().nextInt(partitionCount);
+        targetPartitionId = ThreadLocalRandomProvider.get().nextInt(partitionCount);
```

Comment:
I know your intention is not to change current behaviour, but `ClientReplicatedMapProxy.onInitialize()` doesn't need a thread local random at all. It's just called once.